client/apps/edit/components/content2/section_controls/test/index.test.coffee

```diff
@@ -11,7 +11,7 @@ r =
   simulate: ReactTestUtils.Simulate
   render: ReactTestUtils.renderIntoDocument
 
-describe 'SectionControls', ->
+describe.skip 'SectionControls', ->
```

Comment:
Intentional?

Reply:
The coffeescript version will be replaced in all sections with the new jsx one, so this test file is going to be deleted tomorrow.